cmd/addoperator.go

```diff
@@ -87,7 +88,21 @@ func (p *AddOperatorParams) PreInteractive(ctx ActionCtx) error {
 		return err
 	}
 	if ok {
-		p.jwtPath, err = cli.Prompt("path or url for operator jwt", p.jwtPath, true, cli.PathOrURLValidator())
+		p.jwtPath, err = cli.Prompt("path or url for operator jwt", p.jwtPath, true, func(v string) error {
+			// is it is an URL or path
+			pv := cli.PathOrURLValidator()
+			if err := pv(v); err != nil {
+				// if it doesn't exist - could it be the name of well known operator
+				if os.IsNotExist(err) {
+					wko, _ := FindKnownOperator(v)
+					if wko != nil {
+						return nil
```

Comment:
This does not make sense to me. If `wko` is not nil, then we return `nil` (for no error), but do the same if it is `nil`..

Reply:
it is a validator - there's no error in that case.

Reply:
Ok, but look at the code, this does not make sense. You return the same result (nil) regardless if wko is nil or not.